torch/onnx/symbolic_opset9.py

```diff
@@ -605,10 +605,12 @@ def symbolic_fn(g, input, kernel_size, stride, padding, dilation, ceil_mode):
 
 
 def _avg_pool(name, tuple_fn):
-    @parse_args('v', 'is', 'is', 'is', 'i', 'i')
-    def symbolic_fn(g, input, kernel_size, stride, padding, ceil_mode, count_include_pad):
+    @parse_args('v', 'is', 'is', 'is', 'i', 'i', 'none')
+    def symbolic_fn(g, input, kernel_size, stride, padding, ceil_mode, count_include_pad, divisor_override=None):
         if ceil_mode and input.type().kind() != "CompleteTensorType":
             return _unimplemented(name, "input size not accesible")
+        if divisor_override and divisor_override.node().kind() != 'prim::Constant':
```

Comment:
shouldn't this check if it's None type or something?  This looks like a constant would pass but not actually be passed to the operator.

Reply:
It's similar to ceil_mode. In case if the value will be provided, type().kind() == "ONNX:Constant"

Reply:
I don't see what ceil_mode has to do with it.  @houseroad can you look at this?

Reply:
I checked two cases: divisor_override not provided, divisor_override is provided. And if divisor_override is not provided it will converted to prim::Constant, otherwise it will be ONNX:Constant.

Reply:
what is the meaning of divisor_override? We would like to check the value of divisor_override to determine whether we can express it in ONNX or not. If not, call _unimplemented is the right choice.